samples/apps/copilot-chat-app/documentuploadapp/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.CommandLine;
+using System.Text;
+using System.Text.Json;
+using Microsoft.Identity.Client;
+
+namespace SemanticKernel.Service.DocumentUploadApp;
+
+public static class Program
```

Comment:
Since this is where developers usually start, could you add XMLDoc to the Program class describing what this console app does?